compatibility/build-release-artifacts-windows.ps1

```diff
@@ -37,6 +37,7 @@ function bazel() {
 
 
 bazel shutdown
+bazel clean --expunge
```

Comment:
Do you want to merge the clean --expunge? I guess in this case, that’s the only option since it doesn’t just affect the compat node which we can reset fairly easily.

Reply:
Yes, and then remove it after the nodes were reset. To avoid poisoning the fresh cache with broken items in the local cache of some nodes.